content/en/getting_started/synthetics/_index.md

```diff
@@ -0,0 +1,198 @@
+
+In this example, the browser test homepage shows that a recent test failed and an alert was received via email. Click on the failed test under **Test Results** or in the email. The failed test step is highlighted with a red `x`. Click the failed test to begin troubleshooting. 
+
+The **Errors & Warnings** tab provides a list of javascript and network errors, the **Resources** tab locates the resource providing this status, and the **Traces** tab maps the entirety of the request in seconds. This test failed as the result of a server timeout. The resource, `https://api.shopist.io/checkout.json`, posted the status and the targeted source of the problem is a controller linked to checkout. You have now successfully found the route of the problem.
+
+{{< img src="getting_started/synthetics/browser-test-failure.png" alt="Browser test failure" responsive="true" style="width:100%;">}}
+
+Datadog also has an [APM integration with Synthetics][6] which allows you to go from a test run that failed to the root cause of the issue by looking at the trace generated by that test run. To link browser test results with APM, whitelist the URLs you want the APM integration headers added to. Use * for wildcards: `https://*.datadoghq.com/*`
+
+Alerts triggered with the APM integration populate under **Test Results**.
+
+## Create an API test
+
+[API tests][7] help you monitor your API endpoints and alert you when they are failing or too slow. These checks verify that your applications are responding to requests and meet any conditions you define, such as response time, HTTP status code, and header or body contents. Use the [Datadog API][8] to see the full list.
+
+In this example, an API test is created to ensure your website is constantly up and providing responses in a specific amount of time.
+
+### Configure the request
+
+1. In the Datadog application, hover over **UX Monitoring** in the left hand menu and select **Synthetics Test**. 
+2. In the top right corner, click the **New Test** button. 
+3. Select **API test**.
+4. Define the configuration of your API test:
+    - Add a start URL.
+    - Select **Advance Options** to use custom request headers, authentication credentials, body content, or cookies.
+    - Add one or two tags, like `prod`, to help organize and filter tests.
+    - Select locations for testing.
+    - Click the **Test URL** button.
+
+#### Define your alert conditions
+
+Now that the test is completed, if you did not create any [assertions][9], they are automatically populated. API tests require at least one assertion to be monitored by Datadog. An assertion is defined by a parameter, an optional property, a comparator, and a target value.
+
+In this example, three default assertions are populated when testing the URL:
+```
+When status code is 200
+And header content-type is text/html; charset=utf
+And response time less than 2000 ms
+```
```

Comment:
Can we put a screenshot like https://cl.ly/cb264b701f9c instead?

Reply:
Added screenshot.